packages/ui5-proxy-middleware/test/ui5/middleware.test.ts

```diff
@@ -2,176 +2,234 @@ import * as utils from '../../src/base/utils';
+            });
+            expect(ui5ProxySpy).toBeCalledWith(
+                expect.objectContaining({ proxy: 'http://proxy.example' }),
+                expect.objectContaining({})
+            );
+        });
 
-    test('none', async () => {
-        await getTestServer(config);
-        expect(ui5ProxySpy).toBeCalledWith(
-            expect.objectContaining({}),
-            expect.objectContaining({ secure: true, logLevel: 'info' })
-        );
-    });
+        test('debug', async () => {
+            await getTestServer({
+                ...config,
+                debug: true
+            });
+            expect(ui5ProxySpy).toBeCalledWith(
+                expect.objectContaining({}),
+                expect.objectContaining({ logLevel: 'debug' })
+            );
+        });
 
-    test('proxy', async () => {
-        await getTestServer({
-            ...config,
-            proxy: 'http://proxy.example'
+        test('secure', async () => {
+            await getTestServer({
+                ...config,
+                secure: true
+            });
+            expect(ui5ProxySpy).toBeCalledWith(expect.objectContaining({}), expect.objectContaining({ secure: true }));
         });
-        expect(ui5ProxySpy).toBeCalledWith(
-            expect.objectContaining({ proxy: 'http://proxy.example' }),
-            expect.objectContaining({})
-        );
-    });
 
-    test('debug', async () => {
-        await getTestServer({
-            ...config,
-            debug: true
+        test('secure', async () => {
+            await getTestServer({
+                ...config,
+                secure: false
+            });
+            expect(ui5ProxySpy).toBeCalledWith(expect.objectContaining({}), expect.objectContaining({ secure: false }));
         });
-        expect(ui5ProxySpy).toBeCalledWith(expect.objectContaining({}), expect.objectContaining({ logLevel: 'debug' }));
-    });
 
-    test('secure', async () => {
-        await getTestServer({
-            ...config,
-            secure: true
+        test('directLoad', async () => {
+            const server = await getTestServer({
+                ...config,
+                directLoad: true
+            });
+
+            await server.get('/index.html');
+            expect(injectScriptsMock).toBeCalled();
         });
-        expect(ui5ProxySpy).toBeCalledWith(expect.objectContaining({}), expect.objectContaining({ secure: true }));
     });
 
-    test('secure', async () => {
-        await getTestServer({
-            ...config,
-            secure: false
+    describe('manifest.json options', () => {
+        const config = {
+            ui5: {
+                path: '/resources',
+                url: 'http://ui5.example'
+            }
+        };
+        beforeEach(() => {
+            ui5ProxySpy.mockClear();
         });
-        expect(ui5ProxySpy).toBeCalledWith(expect.objectContaining({}), expect.objectContaining({ secure: false }));
-    });
 
```

Comment:
Do we need a test for a mising `manifest.json`?

Reply:
Good point, missing test added with 4524f436